cmd/register/register_util.go

```diff
@@ -211,6 +214,40 @@ func hydrateIdentifier(identifier *core.Identifier) {
 	}
 }
 
+// TODO: once this code is moved to flytestdlib, refactor to remove this duplicate method.
+func marshalObjToStruct(input interface{}) (*structpb.Struct, error) {
```

Comment:
These aren't enough?

https://github.com/flyteorg/flytestdlib/blob/master/utils/marshal_utils.go

Reply:
If not, do mind adding them there

Reply:
amazing! thank you for pointing this out, done.

Reply:
actually this doesn't work because casted returns true here (incorrectly) https://github.com/flyteorg/flytestdlib/blob/master/utils/marshal_utils.go#L66